cli/src/commands/asset.ts

```diff
@@ -56,13 +56,20 @@ class UploadFile extends File {
 export const upload = async (paths: string[], baseOptions: BaseOptions, options: UploadOptionsDto) => {
   await authenticate(baseOptions);
 
-  const files = await scan(paths, options);
-  if (files.length === 0) {
+  const scanFiles = await scan(paths, options);
+  if (scanFiles.length === 0) {
     console.log('No files found, exiting');
     return;
   }
 
-  const { newFiles, duplicates } = await checkForDuplicates(files, options);
+  let newFiles;
+  let duplicates: Asset[] = [];
+
+  if (options.skipHash) {
+    newFiles = scanFiles;
+  } else {
+    ({ newFiles, duplicates } = await checkForDuplicates(scanFiles, options));
```

Comment:
Why do we need to wrap the whole line around the parenthesis?

Reply:
> Note: The parentheses ( ... ) around the assignment statement are required when using object literal destructuring assignment without a declaration.
{ a, b } = { a: 1, b: 2 } is not valid stand-alone syntax, as the { a, b } on the left-hand side is considered a block and not an object literal according to the rules of [expression statements](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/Expression_statement). However, ({ a, b } = { a: 1, b: 2 }) is valid, as is const { a, b } = { a: 1, b: 2 }.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

It is required because the ambiguous that is describe before.

Reply:
TIL, thank you!